### DISCUSSION TITLE
Email is only for Password reset?

### DISCUSSION THREAD

**Discussion by tri97nguyen** [kimai/kimai#3395] [General]
Hi everyone,

Setting up email in Kimai is only for password reset, or is there any other functions? I can only find Kimai uses email for resetting password so far.

Any help is much appreciated.

**Comment by kevinpapst** [comment]
Yes, so far no other email exists

**Reply by tri97nguyen** [reply]
Thank you your quick reply!

**Comment by mbiolsi** [comment]
Are there any paid plugins that allow us to send html invoices via email? We have the email transport, customer email addresses and even html invoices... just need the option to send it :)

**Reply by kevinpapst** [reply]
No there are not. But I am open for paid work via email.

**Comment by mbiolsi** [comment]
Thanks for the reply! We are new to Kimai and have already purchased 
Translations and Custom Fields. Will probably be purchasing Expenses 
shortly. Afraid we are not able to find custom development at this time 
but we LOVE Kimai and if an email invoice plugin hits the marketplace we 
may very well be interested in it!

Great software. Cheers!

---
Mike Biolsi
_General Manager_

Meet with Me! [3]

315.788.8560
coughlin.co [4]

On 2022-12-06 00:58, Kevin Papst wrote:

> No there are not. But I am open for paid work via email.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you commented.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/kevinpapst/kimai2/discussions/3395#discussioncomment-4319875
[2] 
https://github.com/notifications/unsubscribe-auth/AA52PXRYPJE23CEMOEI2JZDWL3IYBANCNFSM52M7RI4Q
[3] https://meet.coughlin.co/mike/
[4] https://coughlin.co
<hr><br/>
<h2 style="color:green;">Holiday Notice 🎄</h2>
<h3>Please plan ahead as our offices will be <br/><span style="color:red;">closed</span> the week of <span style="color:green;"> DECEMBER 26-30.</span></h3>
